src/app/static/src/app/apiService.ts

```diff
@@ -155,6 +156,11 @@ export class APIService<S extends string, E extends string> implements API<S, E>
         return this._handleAxiosResponse(axios.post(fullUrl, data, config));
     }
 
+    async post(url: string) {
+        const fullUrl = this._buildFullUrl(url);
```

Comment:
We already have a `post` method:  https://github.com/mrc-ide/hint/pull/207/files#diff-0f0781cfcf5e78654409fce90c2c466fR146 Should just be able to change `data: any` to `data?: any`